docs/samples/contact-center/app.js

```diff
@@ -130,14 +130,15 @@ function register() {
         });
         const loginVoiceOptions = agentProfile.loginVoiceOptions;
         agentLogin.innerHTML = '<option value="" selected>Choose Agent Login ...</option>'; // Clear previously selected option on agentLogin.
-        dialNumber.value = '';
-        dialNumber.disabled = true;
+        dialNumber.value = agentProfile.defaultDn ? agentProfile.defaultDn : '';
```

Comment:
Here, we simply update the dial number if it exists (ie for `EXTENSION` and `AGENT_DN` relogin case).
Note that we disable it if the defaultDn is undefined.

I personally felt this approach was better (ie using the defaultDn of the agent config), however if there are some issues here, I am open to changing.